### PR TITLE
Fixes #342, cleanup netns after test suit

### LINKED ISSUE #342: Do we need clean up the the netns after test suit
after test suit will remain lost of tmp netns under `/var/run/netns/`, do we need auto clean those ns?

### PR DESCRIPTION
**Pull request by smarkm**
Fixes #342, cleanup netns after test suit

### CODE REVIEW

**Comment by squeed** [comment]
Makes sense. Can you squash your commits and force-push? Thanks.

**Comment by smarkm** [comment]
Yes @squeed , but how can I squash? just close this then push a new clearly commit ?

**Comment by mars1024** [comment]
> Yes @squeed , but how can I squash? just close this then push a new clearly commit ?

I think git rebase & force-push will help you :)

File: plugins/main/host-device/host-device_test.go [hunk 1]
```diff
@@ -234,6 +234,7 @@ var _ = Describe("base functionality", func() {
 
 	AfterEach(func() {
 		originalNS.Close()
```
Comment (mars1024) [inline plugins/main/host-device/host-device_test.go]: Could you help to change this to `Expect(originalNS.Close()).To(Succeed())` ?

Comment (smarkm) [reply plugins/main/host-device/host-device_test.go]: Yes, I will do it, thank you @mars1024

**Pull request closed without merging**